php/blocks/controls/class-rich-text.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Rich Text control.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Controls;
+
+/**
+ * Class RichText
+ */
+class Rich_Text extends Control_Abstract {
+
+	/**
+	 * Control name.
+	 *
+	 * @var string
+	 */
+	public $name = 'rich_text';
+
+	/**
+	 * Textarea constructor.
+	 *
+	 * @return void
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->label = __( 'Rich Text', 'block-lab' );
+	}
+
+	/**
+	 * Register settings.
+	 *
+	 * @return void
+	 */
+	public function register_settings() {
+		$this->settings[] = new Control_Setting(
```

Comment:
We should also find a way of disabling the "Field Location" setting, or forcing the Editor option, since it simply doesn't work when loaded in the Inspector.

![Screen Shot 2019-05-28 at 8 26 46 am](https://user-images.githubusercontent.com/1097667/58441077-59faf180-8122-11e9-9051-d06e01ffed42.png)

Reply:
Ah, good catch.